README.rst

```diff
@@ -70,9 +70,110 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Windows** users:
 
-Note: You need to have bower installed before you may install the python packages.
+   - Clone the **coala-html** repository
+
+     ::
+
+       $ git clone https://github.com/coala/coala-html.git
+
+   - Download and install the latest version of Node.js from `official website <https://nodejs.org/en/download/>`__.
+
+   - Change the directory to **coala-html** and run following command to install dependencies
+
+     ::
+
+       $ python setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip install coala-bears
+
+   - Install **bower** to build static web pages
+
+     ::
+
+       $ npm install -g bower
+
+
+   - Run **coala-html** to display results from **coala-json**
+
+     ::
+
+       $ coala-html --dir ..\Tests
+
+
+2. Installation instructions for **Ubuntu/Linux** users:
+
+   - Clone the **coala-html** repository
+
+     ::
+
+      $ git clone https://github.com/coala/coala-html.git
+
+   - Install latest version of **Node.js**
+
+     a. For **Ubuntu** users:
+
+        * *Configure Node.js PPA*
+
+          ::
+
+           $ sudo apt-get install curl python-software-properties
+           $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
```

Comment:
doesnt ubuntu have node?  Above a "Node.js PPA" is mentioned, but this is not a PPA.

We should give a PPA install method.

Also for generic Linux, and even Windows, I am now recommending that we use @asdf-vm in our instructions, as it allows installing of node and python and anything else we need.  (not sure about bower / lets check that)

Reply:
This section is meant for Linux users, sorry my bad.
I will change to `Linux` in both the headings where it is written `for Ubuntu/Linux users`.
> PPA install method
Should I change above heading to this one ?

Reply:
And, should I add installation instructions according to `@asdf-vm` in current instructions ?

Reply:
> doesnt ubuntu have node? 

Need to run `sudo apt install nodejs` to install nodejs

Reply:
@jayvdb And, should I add installation instructions according to `@asdf-vm` in current instructions or a provide a link somewhere in instruction either at end or at the beginning of instructions ?

Reply:
A link to official upstream pages is better.

Reply:
I have added link for installing nodejs via asdf-vm. Will it work? or should I add offical nodejs website link for installing it ?